Changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.2.0.0 (November 2021)
```

Comment:
We can keep it three digit version `0.2.0`. The last version was an aberration, because we had to make very minor change.

Reply:
Fixed